concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java

```diff
@@ -775,6 +776,18 @@ public InvokeInstanceResult invokeInstance(InvokeInstanceRequest request) {
         return InvokeInstanceResult.parse(grpcResponse);
     }
 
+    /**
+     * Retrieves the {@link BlockCertificates} for a given block.
+     * Note that, if the block being pointed to is not a product of ConcordiumBFT, then a INVALID_ARGUMENT exception will be thrown.
```

Comment:
Maybe point out that this is the case for blocks that was created before protocol version 6.